src/go/rpk/pkg/cli/cmd/cluster/partitions/cancel.go

```diff
@@ -0,0 +1,97 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newMovementCancelCommand(fs afero.Fs) *cobra.Command {
+	var (
+		node      int
+		noConfirm bool
+	)
+	cmd := &cobra.Command{
+		Use:   "movement-cancel",
+		Short: "Cancel ongoing partitions reconfigurations",
+		Long: `Cancel ongoing partitions reconfigurations happening in the cluster
```

Comment:
Is there a missing end backtick tag here?

Reply:
In Go, a backtick begins a raw (multiline) string, so the closing backtick is a dozenish lines below